.github/workflows/deploy-preview.yml

```diff
@@ -40,6 +40,21 @@ jobs:
           AWS_REGION: ${{ secrets.AWS_REGION }}
           SOURCE_DIR: "./dist"
           DEST_DIR: pr/${{ github.event.pull_request.number }}
+      - name: Capture usage statistics
+        id: stats
+        run: |
+          LAYER_COUNT=$(node scripts/stats.js -c)
+          STYLE_SIZE=$(node scripts/stats.js -s)
+          LAYER_STATS=$(node scripts/stats.js -a)
```

Comment:
Maybe we could compare these statistics to what’s in the main branch and output the diff. Reviewers would get a clearer picture of the PR’s impact, not just a reminder about where the bloat lies.

Reply:
Yeah, I was thinking about how to do that. Perhaps the script could output JSON and then I could have an additional script that compares the two and puts out a markdown table.

Reply:
That sounds reasonable. Alternatively, does the shell script run on platform with the [`paste`](https://linuxcommandlibrary.com/man/paste) command? If so, we could even mush the two files’ lines together directly in the shell.

Reply:
The paste approach sounds conceptually workable though I think it's less good than being able to output diffs and/or percentage changes in a table column.